tests/test_very_basic_operations.py

```diff
@@ -7,5 +8,11 @@ class AddingTwoNumbersTest(unittest.TestCase):
     def test_equality(self):
         self.assertEqual(add(2, 3), 5)
 
+    def test_equality(self):
+        self.assertEqual(add(0, 3), 3)
+
+    def test_equality(self):
```

Comment:
You can't have 3 same methods in one class. Just put all `self.assertEqual` in one method.